client/state/ui/selectors.js

```diff
@@ -90,3 +90,33 @@ export function isPreviewShowing( state ) {
 export function getPreviewUrl( state ) {
 	return state.ui.currentPreviewUrl;
 }
+
+/**
+ * Returns the current layout focus area
+ *
+ * @param  {Object}  state Global state tree
+ * @return {?String}  The current layout focus area
+ */
+export function getCurrentLayoutFocus( state ) {
+	return get( state, 'ui.layoutFocus.current' );
```

Comment:
Fair point. I don't see that error any more, so who knows what I was doing. Removed.